build.rs

```diff
@@ -93,6 +93,7 @@ fn main() {
     println!("cargo:rerun-if-changed=build.rs");
     println!("cargo:rustc-link-search=native={}", out_dir);
     println!("cargo:rustc-env=XLS_DSO_VERSION_TAG={}", DSO_VERSION_TAG);
+    println!("cargo:rustc-env=XLS_DSO_PATH={}", out_dir);    
```

Comment:
Nit: do you mind calling it XLS_DSO_DIRPATH since it's the path to its enclosing directory not to the file itself